packages/atomic/src/components/insight/atomic-insight-interface/e2e/page-object.ts

```diff
@@ -68,42 +72,49 @@ export class InsightInterfacePageObject extends BasePageObject<'atomic-insight-i
     });
   }
 
-  get connectResultLink() {
-    return this.insightResults.getByRole('link', {name: 'Connect'});
+  get videoResultLink() {
+    return this.page.getByTestId('video-insight-result-link');
```

Comment:
Since we're just dealing with a story here, I took the liberty of adding a `data-testid` on the Video result template to make it easier to select results of "type" Videos.